integration-cli/docker_cli_daemon_test.go

```diff
@@ -1825,27 +1825,28 @@ func (s *DockerDaemonSuite) TestDaemonNoSpaceleftOnDeviceError(c *check.C) {
 
 	// create a 2MiB image and mount it as graph root
 	cmd := exec.Command("dd", "of=/tmp/testfs.img", "bs=1M", "seek=2", "count=0")
-	if err := cmd.Run(); err != nil {
-		c.Fatalf("dd failed: %v", err)
-	}
+	out, err := cmd.CombinedOutput()
+	c.Assert(err, checker.IsNil, check.Commentf(string(out)))
+
 	cmd = exec.Command("mkfs.ext4", "-F", "/tmp/testfs.img")
-	if err := cmd.Run(); err != nil {
-		c.Fatalf("mkfs.ext4 failed: %v", err)
-	}
+	out, err = cmd.CombinedOutput()
+	c.Assert(err, checker.IsNil, check.Commentf(string(out)))
```

Comment:
is the "mkfs.ext4 failed:" you removed still useful here? (same as the other ones)